src/components/Article/index.tsx

```diff
@@ -94,6 +99,9 @@ const Article = ({
         hasNoAuthors={!authors || !authors.length}
       />
     )}
+    {version && fileTitle && (
```

Comment:
Not sure this should be in the article template. As, articles are used everywhere.

It makes sense tho to add on `templates/api.tsx`

Reply:
Good call! I've updated `Article` to have an `extraLinks` property and pass the `JsonLink` component into it from `templates/api.tsx`.